diff -ur odbtp-1.1.2.orig/php/ext/odbtp/config.m4 odbtp-1.1.2/php/ext/odbtp/config.m4
--- odbtp-1.1.2.orig/php/ext/odbtp/config.m4	2004-06-20 22:19:46.000000000 +0200
+++ odbtp-1.1.2/php/ext/odbtp/config.m4	2005-01-05 09:35:10.000000000 +0100
@@ -36,15 +36,15 @@
   fi
 
   # --with-odbtp -> add include path
-  PHP_ADD_INCLUDE($ODBTP_DIR/include)
+  PHP_ADD_INCLUDE($ODBTP_DIR)
 
   # --with-odbtp -> chech for library
-  if test ! -f "$ODBTP_DIR/lib/libodbtp.a"; then
-    AC_MSG_ERROR([libodbtp.a not found])
+  if test ! -f "$ODBTP_DIR/libodbtp.so"; then
+    AC_MSG_ERROR([libodbtp.so not found])
   fi
 
   # --with-odbtp -> add library
-  PHP_ADD_LIBRARY_WITH_PATH(odbtp, $ODBTP_DIR/lib, ODBTP_SHARED_LIBADD)
+  PHP_ADD_LIBRARY_WITH_PATH(odbtp, $ODBTP_DIR, ODBTP_SHARED_LIBADD)
 
   AC_DEFINE(HAVE_ODBTP,1,[ ])
   PHP_NEW_EXTENSION(odbtp, php_odbtp.c, $ext_shared)
