diff -ur odbtp-1.1.2.orig/php/ext/Makefile odbtp-1.1.2/php/ext/Makefile
--- odbtp-1.1.2.orig/php/ext/Makefile	2004-06-21 01:39:18.000000000 +0200
+++ odbtp-1.1.2/php/ext/Makefile	2005-01-04 21:27:31.000000000 +0100
@@ -1,17 +1,17 @@
 #
 #  Module specific variables
 #
-MODULE=php_odbtp.so
+MODULE=odbtp.so
 MODULE_OBJS=php_odbtp.o odbtp.o sockutil.o
-MODULE_CFLAGS=-DHAVE_ODBTP -DCOMPILE_DL_ODBTP
+#MODULE_CFLAGS=-DHAVE_ODBTP -DCOMPILE_DL_ODBTP
 # Uncomment the below line and comment the above line to include MSSQL support
-#MODULE_CFLAGS=-DHAVE_ODBTP -DCOMPILE_DL_ODBTP -DODBTP_MSSQL
+MODULE_CFLAGS=-DHAVE_ODBTP -DCOMPILE_DL_ODBTP -DODBTP_MSSQL
 MODULE_INCLUDES=-I../.. -I./odbtp
 
 #
 # PHP specific variables
 #
-PHP_INCROOT=/usr/local/php/include/php
+PHP_INCROOT=/usr/include/php
 PHP_INCLUDES=-I$(PHP_INCROOT) -I$(PHP_INCROOT)/main \
 	-I$(PHP_INCROOT)/TSRM -I$(PHP_INCROOT)/Zend
 
