--- odbtp-1.1.2/Makefile.am.wiget	2005-01-05 14:36:18.865532404 +0100
+++ odbtp-1.1.2/Makefile.am	2005-01-05 14:55:46.469885723 +0100
@@ -1,7 +1,7 @@
 include_HEADERS = odbtp.h
-lib_LIBRARIES = libodbtp.a
-libodbtp_a_SOURCES = odbtp.c odbtp.h odbtpbuild.h sockutil.c sockutil.h
+lib_LTLIBRARIES = libodbtp.la
+libodbtp_la_SOURCES = odbtp.c odbtp.h odbtpbuild.h sockutil.c sockutil.h
+libodbtp_la_LDFLAGS= -release $(VERSION)
 noinst_PROGRAMS = odbtpcli
 odbtpcli_SOURCES = odbtpcli.c
-odbtpcli_LDADD = -lodbtp
-odbtpcli_LDFLAGS = -L.
+odbtpcli_LDADD = libodbtp.la
--- odbtp-1.1.2/configure.in.wiget	2005-01-05 14:36:27.188779303 +0100
+++ odbtp-1.1.2/configure.in	2005-01-05 14:36:38.081793686 +0100
@@ -7,7 +7,7 @@
 
 dnl Checks for programs.
 AC_PROG_CC
-AC_PROG_RANLIB
+AC_PROG_LIBTOOL
 
 dnl Checks for libraries.
 AC_CHECK_FUNC(gethostent, , AC_CHECK_LIB(nsl, gethostent))
